images/bootstrap/Makefile

```diff
@@ -18,7 +18,7 @@ TAG = $(shell date +v%Y%m%d)-$(shell git describe --tags --always --dirty)
 all: build
 
 build:
-	docker build -t $(IMG):$(TAG) .
+	docker build --no-cache -t $(IMG):$(TAG) .
```

Comment:
do we need to do this elsewhere?

Reply:
for go and bazel we explicitly set versions which is not going to be cached I think. Not sure on other images, but might not be related to this PR.